pkg/sdk/action.go

```diff
@@ -54,6 +56,43 @@ func Create(object Object) (err error) {
 	return nil
 }
 
+// Patches provided object on the server with given patch and updates the arg
```

Comment:
`// Patch patches...`? I believe that's Golang's idiomatic way of commenting a function.

Reply:
yeah that sounds in line with https://github.com/kubernetes/client-go/blob/master/deprecated-dynamic/client.go#L61